phpstan.neon

```diff
@@ -1,8 +1,13 @@
 parameters:
-    level: 6
+    level: 9
```

Comment:
Should we use the `max` alias here?

Reply:
the question is whether we would want to automatically bump to level 10 in case phpstan introduces a new higher level (even though I don't see what it would container to be stricter than level 9).
phpstan did that in the past. For a long time, there was no level 9.

Reply:
I think so yes, because I don't think we are going to remember to bump it to level 10, and also because since that library is small, it should not result in a lot of errors. Even if it did, we could still baseline them and hold contributions to a higher standard immediately, instead of letting in more code that needs to be fixed by maintainers.

Reply:
Me too actually, so let's not block this for days. When there is a new level, I think we will be aware and can bump this on all repositories.